ui/tests/acceptance/dashboard-test.js

```diff
@@ -363,9 +363,16 @@ module('Acceptance | landing page dashboard', function (hooks) {
     });
   });
 
-  module('replication and client count card community version', function () {
+  skip('replication and client count card community version', function (hooks) {
```

Comment:
Skipping this test since it will fail when all tests are run locally. We need to have either a component test or a way to bypass the testing binary.